src/db/config.ts

```diff
@@ -50,7 +50,8 @@ if (environment == 'production') {
   const dbDriver: Dialect = 'postgres';
   sequelizeConnection = new Sequelize(dbName, dbUser, dbPassword, {
     host: dbHost,
-    dialect: dbDriver
+    dialect: dbDriver,
+    logging: environment == 'development'
```

Comment:
Only log all db queries in development. This prevents spamming the console when running tests.